workers/social/lib/social/models/invitation.coffee

```diff
@@ -25,7 +25,7 @@ module.exports = class JInvitation extends jraphical.Module
   @set
     permissions     :
       'send invitations'  : ['member']
-      'remove invitation' : ['admin']
+      'remove invitation' : ['member']
```

Comment:
changing a default permission requires maintenance in the db, i would suggest you to leave as is and implement a function like [this](https://github.com/koding/koding/blob/5956d6ca5f804e58452fe2aea548328fdb8acff8/workers/social/lib/social/models/group/index.coffee#L1150) and remove permit from function call signature, finally add that check into application itself. if user has permission to remove an invite do as is, if user does not have permission but regarding invitation has current user as it's inviter, again do that operation, else fail with an error message